app/javascript/widget/components/AgentMessageBubble.vue

```diff
@@ -10,7 +10,7 @@
       <div
         v-dompurify-html="formatMessage(message, false)"
         class="message-content"
-        :class="$dm('text-black-900', 'dark:text-slate-50')"
+        :class="$dm('text-slate-900', 'dark:text-slate-50')"
```

Comment:
We don't need to use the $dm helper now as we moved from the os based dark mode implementation in Tailwind to class based implementation. 

This could be rewritten as 

```
class="message-content text-slate-900 dark:text-slate-50"
```

We should get rid of the $dm usage when we work in the future.